docs/section11/Rule_11-1.md

```diff
@@ -0,0 +1,63 @@
+# Service_Water_Heating - Rule 11-1
+**Schema Version:** 0.0.37  
+
+**Mandatory Rule:** TRUE
+
+**Rule ID:** 11-1
+
+**Rule Description:** "Where a complete service water-heating system exists, the proposed design shall reflect the actual system type. Where a service water-heating system has been designed the service waterheating type shall be consistent with design documents.  Where no service water-heating system exists or has been designed and submitted with design documents but the building will have service water-heating loads, a service water-heating system shall be modeled that matches the system type in the baseline building design"
+
+**Rule Assertion:** Options are PASS/FAIL
+
+**Appendix G Section Reference:** Table G3.1 #11, proposed column, a & b
+
+**Evaluation Context:** custom evaluation context created for this rule of all SWH Distribution system ids located in the proposed, baseline and user RMDs  
```

Comment:
Evaluation context should make it clear if this rule is evaluated once for all SWH Distribution Systems or once for each SWH Distribution System.